swiftglue/IteratorHelpers.swift

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import Swift
+
+fileprivate var _vtable: [TypeCacheKey : SwiftIteratorProtocol_xam_vtable] 
+    = [TypeCacheKey : SwiftIteratorProtocol_xam_vtable]();
+
+fileprivate struct SwiftIteratorProtocol_xam_vtable
+{   
+	fileprivate var func0: (@convention(c)(UnsafeRawPointer, UnsafeRawPointer) -> ())?;
+}
+
+public final class SwiftIteratorProtocolProtocol<T0> : IteratorProtocol
+{
+	private var _xamarinClassIsInitialized: Bool = false;
+	
+    public init()
```

Comment:
Nit: Is spacing here standard for swift? I see 2 and 4 intend?

Reply:
Is there a standard? It's likely because BTfS puts in spaces for indentation. Maybe I should change that to tabs. Maybe I'll open an issue and assign it to the next person who brings this up.